acceptance/tests/invalid_ssl_config.rb

```diff
@@ -21,7 +21,7 @@
 invalid_config_mismatching_keys = {:broker_ws_uri => broker_ws_uri(master),
                                    :ssl_key => ssl_key_file(agent1, 1),
                                    :ssl_ca_cert => ssl_ca_file(agent1),
-                                   :ssl_cert => alt_ssl_cert_file(agent1, 1)}
+                                   :ssl_cert => ssl_cert_file(agent1, 1, use_alt_path=true)}
```

Comment:
Actually to correct myself, ruby 2.0 did introduce named keyword parameters, so If the intent was to use those the code would need to look more like this:

```
def ssl_cert_file(host, client_number, use_alt_path: false)
  # code here
end

ssl_cert_file(agent1, 1, use_alt_path: true)
```

In that pattern use_alt_path is the name of the parameter.